### PR TITLE
fix(domain.operation.update): translate some form labels

### PR DESCRIPTION
**Pull request by antleblanc**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | DTRSD-22021
| License          | BSD 3-Clause

### :bug: Bug Fixes

9741e63 - fix(domain.operation.update): translate some form labels

### :house: Internal

No quality check required.

### :flags: Translations

- [x] TRANS-28782

Signed-off-by: Antoine Leblanc <antoine.leblanc@ovhcloud.com>

### CODE REVIEW

File: packages/manager/apps/web/client/app/domain-operation/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -73,6 +73,9 @@
   "domains_operations_update_choose_action": "Sélectionnez l'opération desirée",
   "domains_operations_update_key_action": "Opération",
   "domains_operations_update_key_memberContactXXX": "Identifiant du membre",
+  "domains_operations_update_key_firstname": "Mettre à jour le prénom avec un contenu valide",
+  "domains_operations_update_key_name": "Mettre à jour le nom avec un contenu validet",
```
Comment (marie-j) [inline packages/manager/apps/web/client/app/domain-operation/translations/Messages_fr_FR.json:77]:
```suggestion
  "domains_operations_update_key_name": "Mettre à jour le nom avec un contenu valide",
```

File: packages/manager/apps/web/client/app/domain-operation/update/domain-operation-update.controller.js [hunk 2]
```diff
@@ -64,7 +64,16 @@ angular.module('App').controller(
...
+                  indexOf(
+                    [
+                      'action',
+                      'memberContactXXX',
+                      'firstname',
+                      'name',
+                      'identificationNumber',
+                    ],
+                    argument.key,
+                  ) !== -1
```
Comment (marie-j) [inline packages/manager/apps/web/client/app/domain-operation/update/domain-operation-update.controller.js:76]: **nitpick** : we could've turn it into an `includes`

**Review by marie-j — CHANGES_REQUESTED** [review]

File: packages/manager/apps/web/client/app/domain-operation/translations/Messages_fr_FR.json (hunk 1 above)
Comment (antleblanc) [reply packages/manager/apps/web/client/app/domain-operation/translations/Messages_fr_FR.json]: Good catch! It's now fixed.

File: packages/manager/apps/web/client/app/domain-operation/update/domain-operation-update.controller.js (hunk 2 above)
Comment (antleblanc) [reply packages/manager/apps/web/client/app/domain-operation/update/domain-operation-update.controller.js]: Thanks, suggestion has been applied.

**Review by marie-j — APPROVED** [review]

**Pull request merged**